pattern-check

```diff
@@ -13,31 +13,70 @@ NORMAL=$(tput sgr0)
 
 
 vim_pattern='(\\S+\\/)?g?\.?(view|l?n?vim?x?|fzf)(diff)?(-wrapped)?$'
-match_tests=(vim Vim VIM vimdiff lvim /usr/local/bin/vim vi gvim view gview nvim vimx fzf .vim-wrapped .nvim-wrapped)
-no_match_tests=( /Users/christoomey/.vim/thing /usr/local/bin/start-vim )
+match_tests=(
+  vim
+  Vim
+  VIM
+  vimdiff
+  lvim
+  /usr/local/bin/vim
+  vi
+  gvim
+  view
+  gview
+  nvim
+  vimx
+  fzf
+  .vim-wrapped
+  .nvim-wrapped
+)
+no_match_tests=(
+  /Users/christoomey/.vim/thing
+  /usr/local/bin/start-vim
```

Comment:
Let me know if you'd like me to comment-out this unit test case in this PR to work around issue #445.